tests/print_benchmarks.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import platform
+import subprocess
+from logging import getLogger
+
+logger = getLogger(__file__)
+
+
+PYTHON_VERSION = platform.python_version()[0:3]
+PYTHON_SYSTEM = platform.system()
+PYTHON_IMPLEMENTATION = platform.python_implementation()
+PYTHON_ARCHITECTURE = platform.architecture()[0]
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="""
+        Given a package name, prints the most recent benchmarks
+        recorded for that package. Mostly used by `tox` to print
+        benchmarking results.
+        """
+    )
+
+    parser.add_argument(
+        "package",
+        help="""
+        The path of the package with benchmarks to print.
+        """,
+    )
+
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    args = parse_args()
+    benchmark_folder = "{}/tests/.benchmarks/{}-{}-{}-{}".format(
+        args.package,
+        PYTHON_SYSTEM,
+        PYTHON_IMPLEMENTATION,
+        PYTHON_VERSION,
+        PYTHON_ARCHITECTURE,
+    )
+    try:
+        subprocess.call(
+            "cat {}/$(ls {} | tail -1)".format(
+                benchmark_folder, benchmark_folder
+            ),
+            shell=True,
+        )
```

Comment:
I'd prefer vanilla python for this. If we ever start testing on windows, it wouldn't work

Reply:
That was really helpful, I didn't know to do that, thank you! 😄